examples/bert/README.md

```diff
@@ -26,8 +26,32 @@ This workflow performs BERT optimization on CPU with Intel® Neural Compressor q
 
 Config file: [bert_inc_ptq_cpu.json](bert_inc_ptq_cpu.json)
 
+#### Run Intel® Neural Compressor quantization with or without accuracy aware tuning
+
+Accuracy aware tuning is one of unique features provided by Intel® Neural Compressor quantization. This feature can be used to solve accuracy loss pain points brought by applying low precision quantization and other lossy optimization methods. Intel® Neural Compressor also supports to quantize all quantizable ops without accuracy tuning, user can decide whether to tune the model accuracy or not. Please check the [doc](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization.md) for more details.
+
+User can decide to tune the model accuracy by setting accuracy `metric` in Intel® Neural Compressor quantization pass config. If not set, accuracy of the model will not be tuned.
+
+```json
+"passes": {
+    "quantization": {
+        "type": "IncQuantization",
+        "config": {
+            "metric": {
+                "name": "accuracy",

```

Comment:
>name [](http://example.com/codeflow?start=17&length=4)

@trajepl  do we need use type or name here?

Reply:
The evaluation function in `custom` can be `latency` or something else. How about to ignore the `accuracy` check? Clear docs may be enough.